Tasks/NugetPublisher/NuGetPublisher.ps1

```diff
@@ -81,26 +81,28 @@ if($connectedServiceName -and $useExternalFeed)
 }
 elseif($feedName -and (-not $useExternalFeed))
 {
+    if (-not [URI]::IsWellFormedUriString($feedName, [UriKind]::Absolute))
+    {
+        throw (Get-LocalizedString -Key "Feed URI is not properly formatted {0}" -ArgumentList $nugetServer)
```

Comment:
I would make this check after we rename the variable.
Reason for that, we eventually will have logic which maps a feed name to a nuget server URL. So when that happens, no reaction will be needed to move your code.